src/components/molecules/Settings/Workspace/Plugin/PluginAccordion/index.tsx

```diff
@@ -19,17 +19,17 @@ export type PluginAccordionProps = {
 
 const PluginAccordion: React.FC<PluginAccordionProps> = ({ className, items }) => {
   const theme = useTheme();
-  return (
+  return items ? (
```

Comment:
just a suggestion of using
`items &&`
instead of ternary operator

Reply:
The reason why I used the ternary operator is to tell it returns `null` explicitly. Kind of developers preference? haha